Doc/whatsnew/3.2.rst

```diff
@@ -1060,15 +1060,15 @@ of nearly equal quantities:
 
 The :func:`~math.erf` function computes a probability integral or `Gaussian
 error function <https://en.wikipedia.org/wiki/Error_function>`_.  The
-complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``:
+complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``::
```

Comment:
Thanks @berkerpeksag :-)  I have a doubt about consistency.  I agree with you that in general using the `doctest` directive and the `SKIP` option is a better choice than using `::`.  But [about consistency the PEP8 says](https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds) that *"Consistency within one module or function is the most important"*.  In this file there is no one directive `doctest`.  To skip an entire test group the `::` are provided.  To skip just one test inside a group, the `# doctest: +SKIP` comment is provided.  That is why I preferred to not use the `doctest` directive in this case.   But if you really want to use the `doctest` directive, I will use it.  Just let me know.  Thank you very much for your time!